### ISSUE TITLE
extension is broken on chrome/firefox

### ISSUE THREAD

**Issue by Fomiller** [Orbytale/BeyondMarkdown#12]
Extension does not seem to be working on chrome and firefox. I tried manual installation for both.

**Comment by Orbytalee** [comment]
Yes. Because dndbeyond had changes, I need to rework some of the selectors. Once dndbeyond is done with their changes (as far as I know, some are still coming), I'll fix it.

Sent with Shortwave <https://www.shortwave.com?utm_medium=email&utm_content=signature&utm_source=aGlicmFnaW1vd0BnbWFpbC5jb20=>

**Comment by Georgi-Stavrev** [comment]
Any chance for an incoming update? Or has life gotten in the way :D
